WcaOnRails/app/views/static_pages/_delegate_row.html.erb

```diff
@@ -18,8 +18,7 @@
       <%= render "shared/user", user: delegate %>
     <% end %>
   </td>
-  <%# Temporary hack - to be removed once https://github.com/thewca/worldcubeassociation.org/issues/2003 is done %>
-  <td><%= t "enums.user.delegate_status.#{delegate.delegate_status == "board_member" ? "delegate" : delegate.delegate_status}" %>
+  <td><%= t "enums.user.delegate_status.#{delegate.delegate_status}" %>
```

Comment:
This reminds me: should we remove the [`enums.user.delegate_status.board_member` key](https://github.com/jfly/worldcubeassociation.org/blob/f2f53c28c3f596b1b6ca3b9c6259ec702f9ed7f8/WcaOnRails/config/locales/en.yml#L306) from en.yml now?

Reply:
Do we have that key elsewhere?
If not, where should I add it?

Reply:
I think this is only used when generating a HTML select to assign a user their delegate status. I don't think you need to add this key anywhere else.

Reply:
Ok!